autogpt_platform/backend/schema.prisma

```diff
@@ -19,45 +19,148 @@ model User {
   updatedAt DateTime @updatedAt
 
   // Relations
-  AgentGraphs                  AgentGraph[]
+  AgentGraphs                  Agent[]
   AgentGraphExecutions         AgentGraphExecution[]
   AgentGraphExecutionSchedules AgentGraphExecutionSchedule[]
   AnalyticsDetails             AnalyticsDetails[]
   AnalyticsMetrics             AnalyticsMetrics[]
   UserBlockCredit              UserBlockCredit[]
+  ConfiguredAgents             ConfiguredAgent[]
+  UserAgents                   UserAgents[]
+
+  // User Group relations
+  groups                 UserGroup[]              @relation("GroupUsers")
+  adminGroups            UserGroup[]              @relation("GroupAdminUsers")
+  UserAccounting         UserAccounting[]
+  UserSubscription       UserSubscription[]
+  Profile                Profile[]
+  StoreListing           StoreListing[]
+  StoreListingSubmission StoreListingSubmission[]
 
   @@index([id])
   @@index([email])
 }
 
+model UserGroup {
+  id        String   @id @default(uuid())
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
+
+  name         String
+  description  String
+  groupIconUrl String?
+
+  users      User[] @relation("GroupUsers")
+  adminUsers User[] @relation("GroupAdminUsers")
+
+  agents       Agent[]
+  Profile      Profile[]
+  StoreListing StoreListing[]
```

Comment:
inconsistent field case (upper vs lower camel case)

Reply:
I'd has looked through and changed the ones I spotted, missed a few by the looks of it.